static/admin/config.yml

```diff
@@ -127,6 +127,69 @@ collections:
                 widget: "string",
                 required: false,
             }
+          - {
+            label: "Certificate of Analysis",
+            name: "certificate_of_analysis",
+            widget: "file",
+            required: false,
+            }
+          - {
+            label: "Donor Plasmid",
+            name: "donor_plasmid",
+            widget: "string",
+            required: false,
+            }
+          - {
+            label: "hPSCreg Certificate",
+            name: "hPSCreg_certificate_link",
+            widget: "string",
+            required: false,
+            } # TODO: double check widget type, file or string?
```

Comment:
Double-checking: should this widget type be a `file` or `string`? It's `file` for disease lines, but it's `string` in the issue ticket.